great_expectations/expectations/regex_based_column_map_expectation.py

```diff
@@ -42,7 +43,24 @@
 logger = logging.getLogger(__name__)
 
 
+@public_api
 class RegexColumnMapMetricProvider(ColumnMapMetricProvider):
+    """Base class for all RegexColumnMapMetrics.
+
+    RegexColumnMapMetric classes inheriting from RegexColumnMapMetricProvider are ephemeral,
+    defined by their `regex` attribute, and registered during the execution of their associated RegexColumnMapExpectation.
+
+    In some cases, subclasses of MetricProvider, such as RegexColumnMapMetricProvider, will already
+    have correct values that may simply be inherited by Metric classes.
+
+    Attributes:
```

Comment:
For some reason the rendering doesn't like `Attributes:`
```suggestion
    Args:
```